### PR TITLE
Feature/breeding

### PR DESCRIPTION
**Pull request by muchendumaina**

### CODE REVIEW

**Comment by riungemaina** [comment]
Remove package.lock
Use yarn

**Comment by riungemaina** [comment]
The colors are not as they should be.

**Comment by riungemaina** [comment]
The widths are too small

File: src/pages/breeding/breeding.jsx [hunk 1]
```diff
@@ -1,4 +1,130 @@
...
 function Breeding() {
-  return <>hello world</>
+  return (
+    <div className={breedingStyles.main}>
+      <div className={breedingStyles.minor}>
+        <div className={breedingStyles.data}>
+          <h1>PREGNANT SOWS</h1>
+          <div className={breedingStyles.dataRows}>
+            <span className={breedingStyles.names}>
+              <h4>NAME</h4>
```
Comment (riungemaina) [inline src/pages/breeding/breeding.jsx:11]: Put this data in a json file

**Review by riungemaina — COMMENTED** [review]
Remove npm package lock

File: src/pages/breeding/breeding.jsx (hunk 1 above)
Comment (muchendumaina) [reply src/pages/breeding/breeding.jsx:11]: There is no data. Those are class names

**Review by riungemaina — APPROVED** [review]

**Pull request merged**